packages/ag-charts-enterprise/src/features/annotations/scenes/textualStartEndScene.ts

```diff
@@ -0,0 +1,264 @@
+import { type AgAnnotationHandleStyles, _ModuleSupport, _Scene, _Util } from 'ag-charts-community';
+
+import { type AnnotationContext, type Coords, type LineCoords } from '../annotationTypes';
+import { convertLine, invertCoords, validateDatumPoint } from '../annotationUtils';
+import type { TextualStartEndProperties } from '../properties/textualStartEndProperties';
+import { DivariantHandle } from '../scenes/handle';
+import { LinearScene } from '../scenes/linearScene';
+
+const { TextMeasurer } = _ModuleSupport;
+const { BBox } = _Scene;
+
+interface AnchoredLayout {
+    alignment: 'left' | 'center' | 'right';
+    placement: 'inside' | 'outside';
+    position: 'top' | 'center' | 'bottom';
+    spacing: number | _Util.Vec2;
+}
+
+export abstract class TextualStartEndScene<Datum extends TextualStartEndProperties> extends LinearScene<Datum> {
+    override activeHandle?: 'start' | 'end';
+
+    public readonly labelLayout: AnchoredLayout = {
+        position: 'center',
+        alignment: 'left',
+        placement: 'inside',
+        spacing: 0,
+    };
+
+    public readonly handleLayout: AnchoredLayout = {
+        position: 'bottom',
+        alignment: 'left',
+        placement: 'outside',
+        spacing: 0,
+    };
```

Comment:
I think we can remove these. My approach with these didn't work. `handleLayout` is not used anywhere as far as I can see, and `labelLayout` can be replaced with just `datum.position` and `datum.alignment` as I did in `TextualPointScene`.